charts/tekton-pipelines/templates/kaniko/pipeline.yaml

```diff
@@ -53,6 +53,10 @@ spec:
           value: $(params.docker_registry)
         - name: environment
           value: "$(params.environment)"
+        {{ if .Values.kaniko.baseDockerImage }}
+        - name: base_docker_image
+          value: "$(params.base_docker_image)"
+        {{ end }}
```

Comment:
```suggestion
        - name: extra_args
          value: $(params.kaniko_extra_vars)
```

Reply:
Fo that we need to declare `kaniko_extra_vars` earlier like here https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/ef989951f14ebc3816f3603305a46f5a5cd18ce7/charts/tekton-pipelines/templates/kaniko/tasks/kaniko.yaml#L45

Reply:
So with only that few changes we can get much more capability to control build workflow.

Reply:
Good point, thank you, @MikhailKorol-saritasa!

Reply:
yep, that's it